ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts

```diff
@@ -0,0 +1,37 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: 0x3B8, // @TODO: change to zoneid entry when available
+  triggers: [
+    {
+      id: 'ToZ Cinduruva Isitva Siddhi',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '62A9', source: 'Cinduruva', capture: true }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'ToZ Sanduruva Isitva Siddhi',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '62C0', source: 'Sanduruva', capture: true }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    // Trash mob tankbuster
+    {
+      id: 'ToZ Wheel',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5E52', source: 'Zot Roader', capture: true }),
```

Comment:
It looks like there's no startsUsing for this attack, it's just an uncasted ability.

Reply:
Hmm, I guess it'll need to be dropped from triggers then. I wanted to include it due to how hard it hits unmitigated but with no startsCasting I'm not sure if there's a feasible way to warn for it.

Reply:
I'll try to generate timelines for this tonight and we'll see whether we can do a timeline trigger instead.

Reply:
That isn’t a boss, and I am sad now. T_T